data/users/externalldap.py

```diff
@@ -83,6 +76,9 @@ def __enter__(self):
             logger.debug("TLS Fallback enabled in LDAP")
             self._conn.set_option(ldap.OPT_X_TLS_TRY, 1)
 
+        # Must come _after_ all other TLS options
+        self._conn.set_option(ldap.OPT_X_TLS_NEWCTX, ldap.OPT_OFF)
```

Comment:
What's the difference between `OPT_X_TLS_TRY` and `OPT_X_TLS_NEWCTX`, and what does `OPT_OFF` do?

Reply:
Try is the "fallback to non-tls".

newctx explained https://github.com/python-ldap/python-ldap/issues/55